programs/ssl/ssl_client2.c

```diff
@@ -1499,7 +1593,7 @@ int main( int argc, char *argv[] )
         else if( strcmp( p, "tickets" ) == 0 )
         {
             opt.tickets = atoi( q );
-            if( opt.tickets < 0 || opt.tickets > 2 )
+            if( opt.tickets < 0 || opt.tickets > 1 )
```

Comment:
What's the rationale for this change?

Reply:
There is only the option to enable and disable the use of tickets for the ssl_client2.c